mlflow/transformers/__init__.py

```diff
@@ -1571,7 +1577,13 @@ def _get_default_pipeline_signature(pipeline, example=None, model_config=None) -
             if _contains_params(example):
                 example, params = example
             example = _format_input_example_for_special_cases(example, pipeline)
-            prediction = generate_signature_output(pipeline, example, model_config, params)
+            prediction = generate_signature_output(
+                pipeline=pipeline,
+                data=example,
+                model_config=model_config,
+                params=params,
+                flavor_config=flavor_config,
```

Comment:
This is a fix for another small bug - flavor_config needs to be passed for inferring signature (otherwise logging an InstructionalPipeline with input_example will raise NPE).